### DISCUSSION TITLE
Disconnect after reading?

### DISCUSSION THREAD

**Discussion by sebska** [wlcrs/huawei_solar#638] [General]
Hi,

i want to use the connection to the Inverter (LAN via SDongle) simultaniously with my "solaranzeige" server.
So: does this Integration disconnects from the port (the inverter) after reading the values? E.g. if i put every 2 minutes it will: connect, read the values, disconnect and wait until 2 minutes are over.
Or does it connect to the converter an keeps the connection endlessly? Then no other connection is possible for other programs/servers..

Regards Sebastian

**Comment by Roving-Ronin** [comment]
@sebska 

Polling is every ~30 seconds and the Huawei sDongle only allows 1 connection.

If wanting more, you need to try use the Modbus proxy add for HA (future EMMA from Huawei looks to support multiple Modbus connections, but will need to see if it works with WLCRS ... cross fingers it should / use existing Modbus registers)
